shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -515,9 +515,6 @@ object GlutenConfig {
       (
         COLUMNAR_VELOX_CONNECTOR_IO_THREADS.key,
         COLUMNAR_VELOX_CONNECTOR_IO_THREADS.defaultValueString),
-      (
-        COLUMNAR_VELOX_SPLIT_PRELOAD_PER_DRIVER.key,
-        COLUMNAR_VELOX_SPLIT_PRELOAD_PER_DRIVER.defaultValueString),
```

Comment:
Can you explain this change? Thanks.

Reply:
Now we need to pass this config to native by `getNativeSessionConf`, so these two lines should be removed.
https://github.com/oap-project/gluten/blob/main/cpp/velox/compute/WholeStageResultIterator.cc#L404